import_install_example.sh

```diff
@@ -1,21 +1,25 @@
 #!/bin/bash
 
+default_library_name='shtdlib.sh'
+default_base_download_url='https://raw.githubusercontent.com/sdelements/shtdlib/master'
```

Comment:
Maybe break out `master` into it's own variable in case you want to test a certain branch?